kubernetes/admission/example_fluxinterval.rego

```diff
@@ -33,9 +35,24 @@ deny[msg] {
 	arg := container.args[j]
 
 	# Extract interval argument value
-	interval := split(arg, "--sync-interval=")[1]
+	split_interval := split(arg, "--sync-interval=")
+	interval := split_interval[1]
+	seconds := convert_to_seconds(interval)
 
 	# Ensure values is not of seconds and is at least 2 digits with minutes
```

Comment:
Nit: Does this comment need to be updated ?

Reply:
fixed